compiler-core/templates/docs-css/index.css

```diff
@@ -527,6 +527,30 @@ body.drawer-open .label-closed {
   margin: 0 0 0 var(--small-gap);
 }
 
+.visibility-tag {
+  background-color: var(--bg-shade-2); 
+  color: var(--text); 
+  padding: 2px 6px; 
+  border-radius: 4px; 
+  border-style: solid;
+  border-width: 1px;
+  border-color: var(--fg-shade-2);
+  font-size: 0.9em; 
+  margin-left: 8px;
+  float: right;
+}
+
+.visibility-tag:hover::after {
+  content: attr(title);
+  display: block;
+  position: absolute;
+  background-color: var(--bg-shade-3);
+  color: var(--fg-shade-3);
+  padding: 6px; 
+  border-radius: 2px;
+  margin-top: 6px; 
+}
```

Comment:
What's this bit for? Seem unused, and I think it replicates what the title attribute does by default?

Reply:
Ah, that's a leftover from myself playing around with css (haven't done frontend stuff in quite some time).

Will remove shortly ❤️